tracer/src/Datadog.Trace/Logging/LogsInjection/LogEnricher.cs

```diff
@@ -25,21 +25,21 @@ public LogEnricher(ILogProvider logProvider)
             _logProvider = logProvider;
         }
 
-        public void Initialize(IScopeManager scopeManager, string defaultServiceName, string version, string env)
+        public void Initialize(string defaultServiceName, string version, string env)
         {
             _versionProperty = version;
             _environmentProperty = env;
             _serviceProperty = defaultServiceName;
-            _traceIdProperty = CreateTracerProperty(scopeManager, t => t.Active?.Span.TraceId.ToString());
-            _spanIdProperty = CreateTracerProperty(scopeManager, t => t.Active?.Span.SpanId.ToString());
+            _traceIdProperty = CreateTracerProperty(() => Tracer.Instance.DistributedSpanContext?[HttpHeaderNames.TraceId]);
+            _spanIdProperty = CreateTracerProperty(() => Tracer.Instance.DistributedSpanContext?[HttpHeaderNames.ParentId]);
```

Comment:
Using `Tracer.Instance` in here is problematic for testing 🙁 It will likely cause some other tests to break I think - that's why we started passing in `IScopeManager` EDIT: I see you used `[TracerRestore]` - maybe that'll be enough!

Reply:
The tests seem to pass so I think we're all good 😄